CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to the Virtual Rainforest
+
+We're really happy that you are thinking about contributing to the Virtual Rainforest
+project. The whole point of the project is to generate a tool that can be used widely by
+the community and the best way for that to happen is for the community to build it.
+
+To make contributing as seemless as possible, please note these developer guidelines.
+
+## Development setup
+
+Further notes are available [here](source/docs/develop/developer_setup.md) but - once
+you have cloned the Virtual Rainforest repository to your own machine - you will need to
+set up our development toolchain on your machine to contribute to the project.
+
+* We use `poetry` ([](https://https://python-poetry.org/)) to manage the package
+  development. Once you have installed `poetry`, you can use `poetry install` within
+  the repo to start using it.
+* We use `pre-commit` ([](https://pre-commit.com/)) to maintain code quality on
+  submitted code. Before changing the code, install `pre-commit` and then use
+  `pre-commit` install to make sure that your code is being checked.
+
+## Contributing code
+
+* Code contributed to the Virtual Rainforest **must** address a documented issue on the
+  [issue tracker](https://github.com/ImperialCollegeLondon/virtual_rainforest/issues).
```

Comment:
I think this requirement is a bit too hard: a user may want to contribute a small one-line fix and making them jump through extra hoops might be offputting.

Again though, it's your project, so up to you!